docs/library/network.WIZNET5K.rst

```diff
@@ -56,16 +56,6 @@ Methods
    Returns ``True`` if the physical Ethernet link is connected and up.
    Returns ``False`` otherwise.
 
-.. method:: WIZNET5K.ifconfig([(ip, subnet, gateway, dns)])
```

Comment:
This is being removed because it's documented in the AbstractNIC class, right?

Maybe there needs to be some note in this file that this class does extend the AbstractNIC type and so does have the ifconfig/ipconfig methods.

Reply:
Yes, also there is also `.active`, and `.status` neither are directly documented here and only implicitly inherited. Interestingly the wiznet5k-class does not implement all functions from AbstractNIC (like `.scan`) but even before the change it implemented more than documented here.

To make it kind-of consistent we could remove the docs for `WIZNET5K.isconnected` and then add a paragraph linking to `AbstractNIC` for: isconnected, active, ifconfig (maybe leave that out to encourage ipconfig instead), ipconfig, config and status.

Reply:
> To make it kind-of consistent we could remove the docs for `WIZNET5K.isconnected` and then add a paragraph linking to `AbstractNIC` for: isconnected, active, ifconfig (maybe leave that out to encourage ipconfig instead), ipconfig, config and status.

Yes.  But that sounds like something for a separate PR.